packages/dds/tree/src/simple-tree/api/treeApiBeta.ts

```diff
@@ -91,11 +102,63 @@ export const TreeBeta = {
 	 * @returns A callback function which will deregister the event.
 	 * This callback should be called only once.
 	 */
+	on<K extends keyof TreeChangeEventsBeta<TNode>, TNode extends TreeNode>(
+		node: TNode,
+		eventName: K,
+		listener: NoInfer<TreeChangeEventsBeta<TNode>[K]>,
+	): () => void;
+
+	/**
+	 * Clones the persisted data associated with a node. Some key things to note:
+	 * - Local state, such as properties added to customized schema classes, will not be cloned. However, they will be
+	 * initialized to the state after running the constructor, just like if a remote client had inserted the same node.
```

Comment:
Might be simpler to avoid mentioning remote clients here?
```suggestion
	 * initialized to their default state just as if the node had been created via its constructor.